### PR TITLE
Redesign Libraries section

### PR DESCRIPTION
**Pull request by LidorLapid**

### CODE REVIEW

File: assets/scss/partials/elements/_strip.scss [hunk 1]
```diff
@@ -16,6 +22,45 @@
...
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    @include breakpoint(large) {
+      display: block;
+    }
+  }
```
Comment (wellhairy) [inline assets/scss/partials/elements/_strip.scss]: Due to new implementation of the picture element i'll be removing these along with the picture element work pending benchmarking of the picture element.

File: assets/scss/partials/structure/_nav.scss [hunk 2]
```diff
@@ -70,24 +73,24 @@
...
     margin-bottom: $margin-bottom;
+    padding-bottom: 15px;
+    border-bottom: solid 4px $kingfisher-teal;
   }
 
   &--white-background {
     @include breakpoint(large) {
       #{$this}__list {
-        padding: 15px;
+        padding: 30px 15px 12px;
```
[the remark this answers is not in the document]
Comment (LidorLapid) [reply assets/scss/partials/structure/_nav.scss]: That page doesn't seem to be using this implementation -- I can't see a difference when applying this CSS.

File: assets/scss/partials/_variables.scss [hunk 3]
```diff
@@ -72,5 +74,8 @@ $gutter: 30px;
 $aspect-ratio-square: 100%; // 100 : 100 also 1 : 1 or 100%
 $aspect-ratio-photo: 100% / 3 * 2; // 1920 : 1280 also 1.5 : 1 or 66.66%
 $aspect-ratio-widescreen: 100% / 16 * 9; // 1920 : 1080 also 1.78 : 1 or 56.25%
-$aspect-ratio-super-widescreen: 100% / 18 * 9; // 1920 : 960 also 2 : 1 or 50%
 $aspect-ratio-cinema-widescreen: 100% / 2.35; // 1920 : 817 also 2.35 : 1 or 42.5532%
+
+$one-over-two: 100% / 2 * 1;
+$one-over-three: 100% / 3 * 1;
+$one-over-four: 100% / 4 * 1;
```
[the remark this answers is not in the document]
Comment (wellhairy) [reply assets/scss/partials/_variables.scss]: or maybe we should get rid of the existing ones in favour for something closer to these? I don't mind what approach we use but maybe we should discuss this further.

File: assets/scss/partials/elements/_strip.scss [hunk 4]
```diff
@@ -25,6 +50,7 @@
     left: 0;
     background-position: center center;
     background-size: cover;
+    object-fit: cover;
```
[the remark this answers is not in the document]
Comment (wellhairy) [reply assets/scss/partials/elements/_strip.scss]: This seems like it was left over from when the implementation was slightly different. I'll go ahead and remove it.

File: assets/scss/partials/structure/_media.scss [hunk 5]
```diff
@@ -44,3 +40,55 @@ picture img {
 img[src*="doubleclick"] {
   display: none;
 }
+
+.picture {
```
[the remark this answers is not in the document]
Comment (wellhairy) [reply assets/scss/partials/structure/_media.scss]: agreed

**Pull request merged**